### DISCUSSION TITLE
Style the Library/History/Bookmarks-Window

### DISCUSSION THREAD

**Discussion by JVariance** [Godiesc/firefox-one#17] [Ideas]
It would be cool to have the library window also styled in the modular and rounded look.

**Comment by Godiesc** [comment]
hi, I will try.

**Comment by Godiesc** [comment]
the style was made some time ago:

![imagen](https://github.com/Godiesc/firefox-one/assets/22057609/5567dbe7-a300-4625-9789-8f0c1ec63340)
